devel/uboot-tools/Portfile

```diff
@@ -17,20 +21,22 @@ long_description    U-Boot is a boot loader for Embedded boards based on \
                     port compiles some binary tools that can be used to \
                     deploy u-boot on embedded devices.
 
-checksums           rmd160  124de3a5355915ccb2291b4a01f21e7710290521 \
-                    sha256  616b446e15d1cd1ab6461ebb61ac6655a2b13e902fe0601f36c4affb3949d416 \
-                    size    19630685
+checksums           rmd160  25357e70695de9aac1cee2a752cb4fd82def66ed \
+                    sha256  0c022ca6796aa8c0689faae8b515eb62ac84519c31de3153257a9ee0f446618f \
+                    size    20176030
 
 depends_build       port:gmake
 
 depends_lib         path:lib/libssl.dylib:openssl
 
+compiler.c_standard 2011
```

Comment:
Should you remove this line as well?

Reply:
So, that is not needed to fix the issue with `strndup`?

Reply:
Probably not. That's what @ryandesign suggests.

> I still don't see why you're setting configure.c_standard to 2011 or adding -std=gnu11. So far, it just looks like you need the compatibility implementation of strndup from legacy support and that's it.

Reply:
Remove the `compiler.c_standard 2011` line unless you can justify why this port would require a C11-compatible compiler. `strndup` has nothing to do with C11.

Reply:
Ah no, keep it: I see that the project's Makefile says:
```
# With the move to GCC 6, we have implicitly upgraded our language
# standard to GNU11 (see https://gcc.gnu.org/gcc-5/porting_to.html).
# Some Linux distributions (including RHEL7, SLES13, Debian 8) still
# have older compilers as their default, so we make it explicit for
# these that our host tools are GNU11 (i.e. C11 w/ GNU extensions).
CSTD_FLAG := -std=gnu11
```